src/main/java/core/basesyntax/service/FruitShopService.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UnknownOperationException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FruitShopService {
+    private Map<String, Integer> fruitInventory = new HashMap<>();
```

Comment:
The `fruitInventory` map should likely be encapsulated better. As it stands, it's mutable from outside the class because the `getInventory` method is returning the original map. Consider returning a copy of the map or an unmodifiable view to adhere to the encapsulation principle.